code/game/turfs/simulated/floor/transparent.dm

```diff
@@ -0,0 +1,44 @@
+/turf/simulated/floor/transparent/glass
+	name = "glass floor"
+	desc = "Don't jump on it, or do, I'm not your mom."
+	icon = 'icons/turf/floors/glass.dmi'
+	icon_state = "unsmooth"
+	baseturf = /turf/space
+	broken_states = list("damaged1", "damaged2", "damaged3")
+	smooth = SMOOTH_MORE
+	canSmoothWith = list(/turf/simulated/floor/transparent)
+	footstep = FOOTSTEP_PLATING
```

Comment:
This NEEDS custom footstep sounds.

Being a coder sometimes involves finding some overly eager guy who's willing to work you have no experience in for you.